Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1139,19 +1146,27 @@ class BrowserViewController: UIViewController {
 
             TabEvent.post(.didChangeURL(url), for: tab)
         }
-
-        if tab !== tabManager.selectedTab, let webView = tab.webView {
-            // To Screenshot a tab that is hidden we must add the webView,
-            // then wait enough time for the webview to render.
-            view.insertSubview(webView, at: 0)
-            // This is kind of a hacky fix for Bug 1476637 to prevent webpages from focusing the
-            // touch-screen keyboard from the background even though they shouldn't be able to.
-            webView.resignFirstResponder()
-
-            DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(500)) {
-                self.screenshotHelper.takeScreenshot(tab)
-                if webView.superview == self.view {
-                    webView.removeFromSuperview()
+        
+        // Represents WebView observation or delegate update that called this function
+        switch webViewStatus {
+        // If change in title or url occurred, do not need to take a new screenshot
+        case .title, .url:
+            return
+        // If navigation occurred, take new screenshot
+        case .finishedNavigation:
+            if tab !== tabManager.selectedTab, let webView = tab.webView {
```

Comment:
It's possible we should just remove taking screenshot from here entirely, including for when this is called from `webView(_ webView: WKWebView, didFinish navigation: WKNavigation!)`

It doesn't seem necessary to take a screenshot when the tab page reloads the webviews, and this can result in us capturing a screenshot of a reloading webview, which doesn't seem much better than a blank screenshot.  I tested removing all of this screenshot code from this function and couldn't identify any failures in the functioning of the app or the tabs screen. Could be a performance improvement as well. Thoughts @garvankeeley ?